pkg/cmdsetup/data.go

```diff
@@ -49,7 +52,9 @@ func newDataCmd(runners ...run.Unit) *cobra.Command {
 	metricSvc := observability.NewMetricService(metaSvc, metricsPipeline, "data", nil)
 	pm := protector.NewMemory(metricSvc)
 	pipeline := sub.NewServer(metricSvc)
-	propertySvc, err := property.NewService(metaSvc, pipeline, metricSvc, pm)
+	client := pub.New(metaSvc, databasev1.Role_ROLE_DATA)
+	messenger := gossip.NewMessenger(metricSvc, client, pipeline)
```

Comment:
The gossip server should be initialized within the `property` module, as it is not a top-level module.